catalyst-toolbox/src/rewards/voters.rs

```diff
@@ -15,6 +69,8 @@ pub enum Error {
     MultipleEntries,
     #[error("Unknown voter group {0}")]
     UnknownVoterGroup(String),
+    #[error("hash is not a valid blake2b256 hash")]
```

Comment:
"Not a valid blake2b256 hash" seems better ))

Reply:
"Invalid blake2b256 hash" would be my proposal.